packages/view-types/spatial-three/src/three-utils.js

```diff
@@ -361,15 +361,29 @@ function setUniformsTextures(
 * @param textures         ... from Store
 * @param contrastLimits   ... given by UI
 * @param volumeMinMax     ... from Store
-* @param scale            ... from Store
+* @param scaleOrUndefined ... from Store
 */
 export function create3DRendering(volumes, channelTargetC, channelsVisible, colors, textures,
-  contrastLimits, volumeMinMax, scale, renderstyle, layerTransparency,
+  contrastLimits, volumeMinMax, scaleOrUndefined, renderstyle, layerTransparency,
   xSlice, ySlice, zSlice, originalScale) {
   const texturesList = [];
   const colorsSave = [];
   const contrastLimitsList = [];
   let volume = null;
+  let scale = scaleOrUndefined;
```

Comment:
This was to resolve a linting error https://eslint.org/docs/latest/rules/no-param-reassign